disttask/importinto/dispatcher.go

```diff
@@ -350,7 +350,7 @@ func (h *flowHandle) updateCurrentTask(task *proto.Task) {
 	if h.currTaskID.Swap(task.ID) != task.ID {
 		taskMeta := &TaskMeta{}
 		if err := json.Unmarshal(task.Meta, taskMeta); err == nil {
-			h.disableTiKVImportMode.Store(taskMeta.Plan.DisableTiKVImportMode)
+			h.disableTiKVImportMode.Store(taskMeta.Plan.DisableTiKVImportMode || taskMeta.Plan.IsRaftKV2)
```

Comment:
why we will disable it?

Reply:
for raftkv2, switch mode in local backend

Reply:
maybe leave a comment here

Reply:
[3806d90](https://github.com/pingcap/tidb/pull/45482/commits/3806d90b9350d6248f375a39a60f4b9d85e80d4c)